doc/planning_scene_ros_api/src/planning_scene_ros_api_tutorial.cpp

```diff
@@ -178,14 +192,14 @@ int main(int argc, char** argv)
   // attached objects or collisions objects by clearing those fields
   // first.
   /* Carry out the REMOVE + ATTACH operation */
-  ROS_INFO("Attaching the object to the hand and removing it from the world.");
+  RCLCPP_INFO(LOGGER, "Attaching the object to the hand and removing it from the world.");
   planning_scene.world.collision_objects.clear();
   planning_scene.world.collision_objects.push_back(remove_object);
   planning_scene.robot_state.attached_collision_objects.push_back(attached_object);
   planning_scene.robot_state.is_diff = true;
-  planning_scene_diff_publisher.publish(planning_scene);
+  planning_scene_diff_publisher->publish(planning_scene);
 
-  visual_tools.prompt("Press 'next' in the RvizVisualToolsGui window to continue the demo");
+  prompt("Press 'next' in the RvizVisualToolsGui window to continue the demo");
```

Comment:
Do you want to change the prompt prints to not include `RvizVisualToolsGui`?

Reply:
They were actually not working as intended for me, so I used the normal function one.

Reply:
The last time I checked rviz_visual_tools GUI had not been ported to ROS2.

In the other tutorials I ported I commented out the `visual_tools.prompt` and added my own to make it easy to re-enable when it gets added back in. It's up to you if you want to do the same approach here...
```
prompt("Press 'Enter' to start the demo");
/* visual_tools.prompt("Press 'next' in the RvizVisualToolsGui window to start the demo"); */
```

Reply:
Yeah sure, I can make it that way.